tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/runtime/api/RuntimeAPITest.java

```diff
@@ -39,10 +39,10 @@ public void testRuntimeAPIs(String packageName) {
     @DataProvider
     public Object[] packageNameProvider() {
         return new String[]{
+                "async",
```

Comment:
Do we need to create a issue to check why this fails if the order is changed?

Reply:
CReated https://github.com/ballerina-platform/ballerina-lang/issues/37867